nixos/mixins/systemd-boot.nix

```diff
@@ -5,7 +5,6 @@
   #boot.loader.efi.canTouchEfiVariables = true;
 
   # Use systemd-boot to boot EFI machines
-  boot.loader.systemd-boot.configurationLimit = 10;
```

Comment:
I believe we need to either restore this line as a generic default for non-servers or explicitly set it in the desktop profile. Otherwise all non-servers fall back to nixpkgs default of `null` instead of the intended `10` generations.

Reply:
Yup. Restored to 10 with a lower priority.